02-dev_apps/first-actor/buildsign.md

```diff
@@ -0,0 +1,81 @@
+# Signing your actor
+
+First off, lets build and sign the actor as-is. This way, we can see if the module is behaving as we expect.
+
+<details>
+  <summary>Rust</summary>
+  Build it    
+  
+  `cargo build --release`{{execute}}   
+  
+  Sign it    
+  `wash claims sign target/wasm32-unknown-unknown/release/calculator.wasm --http_server --name "calculator" --ver 0.1.0 --rev 0`{{execute}}
+</details>
+<details>
+  <summary>Go</summary>
+
+Build it  
+ `mkdir -p build && tinygo build -o build/calculator.wasm -target wasm -no-debug main.go`{{execute}}
+
+Sign it  
+ `wash claims sign build/calculator.wasm --http_server --name "calculator" --ver 0.1.0 --rev 0`{{execute}}
+
+</details>
+<details>
+  <summary>AssemblyScript</summary>
+  
+  Build it    
+  `npm run build`{{execute}}
+
+Sign it  
+ `wash claims sign calculator.wasm --http_server --name "calculator" --ver 0.1.0 --rev 0`{{execute}}
```

Comment:
for some reason "sign it" shows up on the same line as the build it script, is that due to indents? (not a big deal)